blog/steering/[BLOG]-toc-2021-election-announcement.md

```diff
@@ -0,0 +1,41 @@
+It’s time to kick off the Knative Technical Oversight Committee (TOC) elections
+again for 2021, and we are ready to accept nominations!
+
+This election will shape the future of Knative as a community and project. While
+WGs help shape the individual direction of the project, the [TOC covers the
+technical direction, health of the project and community as a
+whole](https://github.com/knative/community/blob/main/TECH-OVERSIGHT-COMMITTEE.md#charter).
+There are 2
+seats up for election and each of those seats will be held for two year terms.
+
+Right now, there are 2 things we need each Knative community member to do:
+
+1) If you want to stand for election, we have [detailed instructions for the
+nomination
+process](https://github.com/knative/community/tree/main/elections/2021-TOC#candidacy-process).
+2) If
+you want to vote in the election, please read the [eligibility
+criteria](https://github.com/knative/community/blob/main/mechanics/TOC.md#voter-eligibility). If you think you are eligible to vote, please make sure that your GitHub id appears
+in the voters.yaml file, and if not, please use the [voter exception
+process](https://github.com/knative/community/tree/main/elections/2021-TOC)
+in our voting tool, Elekto.
+
+Here are a few important deadlines to keep in mind:
+
+- April 23: All candidate nominations and voting exceptions due
+- April 27: Election Begins
+- May 11: Election Closes
+- May 13: Announcement of Results
+
+For this election, we will be trying a new voting tool called Elekto.
+Commissioned by the CNCF for the Kubernetes project, Elekto has the advantage of
+not requiring any email for voting, just your Github login.  However, this will
+be the first time a project other than Kubernetes is using Elekto, so please be
+prepared for some hiccups.
+
+For more details, please visit the [TOC election](
+https://github.com/knative/community/tree/main/elections/2021-TOC) document in
+our GitHub repository.
+
+Dawn Foster and Josh Berkus, TOC Election Officers
```

Comment:
Maybe add the elections email group in case people want to reach out?

Reply:
Done, thanks!